### PR TITLE
docs: add "Improved Patch Delivery" blog

### PR DESCRIPTION
**Pull request by felangel**
## Status

**READY**

## Description

- docs: add "Improved Patch Delivery" blog

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [X] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: src/pages/blog/improved-patch-delivery.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+We're excited to announce some improvements to our cloud infrastructure which
+improve our patch delivery, availability, and performance around the world 🥳
+
+## What's Changed
+
+When a patch is published via `shorebird patch`, the generated patch artifact is uploaded to Shorebird's cloud storage.
+On app launch, the corresponding release running on users' devices will ping Shorebird's API to check whether a newer patch is available.
+If a newer patch is available, Shorebird's API will respond with a link to the patch artifact and the patch is then downloaded and installed on the device.
+
+We've made some improvements to how we store and serve patch artifacts to both improve availability (e.g patches can be downloaded by devices in more places around the world) and performance (e.g patches take less time to download).
```
Comment (bryanoltman) [inline src/pages/blog/improved-patch-delivery.md:19]:
```suggestion
We've made some improvements to how we store and serve patch artifacts to both improve availability and performance—patches can be downloaded faster in more places around the world.
```

File: src/pages/blog/improved-patch-delivery.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+layout: ../../layouts/BlogLayout.astro
+title: Blog | Improved Patch Delivery
+description: Announcing improved global patch delivery, availability, and performance.
+date: 9/5/2024
+---
+
+![Improved patch delivery header image](../../assets/images/blog/improved-patch-delivery/heading.png)
+
+We're excited to announce some improvements to our cloud infrastructure which
+improve our patch delivery, availability, and performance around the world 🥳
```
Comment (eseidel) [inline src/pages/blog/improved-patch-delivery.md:11]:
```suggestion
We're excited to announce some improvements to cloud infrastructure which
improve patch delivery, availability, and performance around the world 🥳
```

File: src/pages/blog/improved-patch-delivery.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## What's Changed
+
+When a patch is published via `shorebird patch`, the generated patch artifact is uploaded to Shorebird's cloud storage.
```
Comment (eseidel) [inline src/pages/blog/improved-patch-delivery.md:15]:
```suggestion
When a patch is published via `shorebird patch`, the generated patch artifact is uploaded to Shorebird's cloud.
```

File: src/pages/blog/improved-patch-delivery.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+On app launch, the corresponding release running on users' devices will ping Shorebird's API to check whether a newer patch is available.
+If a newer patch is available, Shorebird's API will respond with a link to the patch artifact and the patch is then downloaded and installed on the device.
+
+We've made some improvements to how we store and serve patch artifacts to both improve availability (e.g patches can be downloaded by devices in more places around the world) and performance (e.g patches take less time to download).
+
+## Wider Availability Worldwide
```
Comment (bryanoltman) [inline src/pages/blog/improved-patch-delivery.md:21]:
```suggestion
## Wider Availability
```

File: src/pages/blog/improved-patch-delivery.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+We've made some changes to our cloud infrastructure to enable Shorebird to deliver patches to more regions around the world. You may now find that users are able to download patches in regions that were previously unavailable.
+
+![China Patch availability image](../../assets/images/blog/improved-patch-delivery/china-availability.png)
+
+**Note: There may still be regions in which Shorebird is unavailable. Please [contact us](mailto:contact@shorebird.dev) if you notice users having trouble downloading patches and we're happy to help.**
+
+## Better Performance Worldwide
```
Comment (bryanoltman) [inline src/pages/blog/improved-patch-delivery.md:29]:
```suggestion
## Faster Downloads
```

**Review by bryanoltman — APPROVED** [review]

File: src/pages/blog/improved-patch-delivery.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+We've made some improvements to how we store and serve patch artifacts to both improve availability (e.g patches can be downloaded by devices in more places around the world) and performance (e.g patches take less time to download).
+
+## Wider Availability Worldwide
+
+We've made some changes to our cloud infrastructure to enable Shorebird to deliver patches to more regions around the world. You may now find that users are able to download patches in regions that were previously unavailable.
+
+![China Patch availability image](../../assets/images/blog/improved-patch-delivery/china-availability.png)
+
+**Note: There may still be regions in which Shorebird is unavailable. Please [contact us](mailto:contact@shorebird.dev) if you notice users having trouble downloading patches and we're happy to help.**
```
Comment (eseidel) [inline src/pages/blog/improved-patch-delivery.md:27]:
```suggestion
```

Comment (eseidel) [reply src/pages/blog/improved-patch-delivery.md:27]: I don't think this adds much.

**Pull request merged**